tsl/src/chunk.h

```diff
@@ -8,17 +8,36 @@
 
 #include <postgres.h>
 #include <fmgr.h>
-
 #include <chunk.h>
 
+/* To track a chunk move or copy activity */
+typedef struct ChunkCopyData
```

Comment:
So, I think we want to embed the FormData in this struct since they overlapped a lot in contents. This is also the typical pattern we have for data structures we persist to metadata tables, like `Hypertable` or `Dimension`.

As pointed out in the previous PR, the FormData is defined in catalog.h but is currently not used for anything.

Reply:
So the idea to remove it completely?